docs/1.2-DRAFT/data-entities.md

```diff
@@ -378,15 +385,13 @@ These can be included for File Data Entities as additional metadata, regardless
     "subjectOf": {"@id": "http://example.com/reports/2019/annual-survey.html"}
   },
   {
-    "@id": "https://zenodo.org/record/3541888/files/ro-crate-1.0.0.pdf",
-    "@type": "File",
-    "name": "RO-Crate specification",
-    "encodingFormat": "application/pdf",
-    "identifier": "https://doi.org/10.5281/zenodo.3541888",
-    "url": "https://zenodo.org/record/3541888"
+    "@id": "http://example.com/reports/2019/annual-survey.html",
```

Comment:
Agree the `.pdf` should not be in here, but I added the entity for the `WebPage` landing page as referenced above.